lib/modules/showImages.js

```diff
@@ -1118,13 +1118,17 @@ function generateIframe(options) {
 	const iframeNode = document.createElement('iframe');
 	iframeNode.src = (module.options.autoplayVideo.value && options.embedAutoplay) ?
 		options.embedAutoplay : options.embed;
-	iframeNode.width = options.width || '640';
-	iframeNode.height = options.height || '360';
+	iframeNode.style.minWidth = options.width || '640px';
+	iframeNode.style.height = options.height || '360px';
 	iframeNode.allowFullscreen = true;
 	iframeNode.setAttribute('frameborder', '0');
-	iframeNode.style.maxWidth = options.maxwidth || '100%';
 
 	iframeNode.expand = () => {
+		iframeNode.dispatchEvent(new CustomEvent('resize', { bubbles: true }));
+		if (iframeNode.style.minWidth.match(/%$/) && iframeNode.clientWidth < 480) {
```

Comment:
Actually, could we accomplish the same thing by doing:
```js
iframeNode.style.minWidth = '480px';
iframeNode.style.width = options.width || '640px';
```
above?

Or is there a reason that `options.width` needs to be the `minWidth`?